usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -95,11 +98,35 @@ def post(self, request):
 
         db_results, values = self.database_data_layer()
 
-        # build response
-        for result in db_results:
-            result["time_period"] = {}
-            for period in values:
-                result["time_period"][self.groupings[period]] = str(result[period])
-                del result[period]
-
-        return Response({"group": self.groupings[self.group], "results": db_results})
+        # Populate all possible periods with no new awards for now
+        hashed_results = {}
```

Comment:
This could be a ordered_dict so the final results don't need to be sorted at the end of the business logic